web/thesauruses/python/control_structures.json

```diff
@@ -39,10 +40,13 @@
       "code": "expression_if_true if condition else expression_if_false"
     },
     "while_loop": {
-      "code": "while condition:\n    statements"
+      "code": "while condition:\n    statements\n\nwhile True:\n    statements\n    if(conditional):\n        break"
```

Comment:
Can you add a `"comment"` here describing the two different scenarios you have since it's not technically one code block?